src/test/java/uk/gov/companieshouse/filinghistory/consumer/mapper/pretransform/ResolutionNodeMapperTest.java

```diff
@@ -0,0 +1,142 @@
+package uk.gov.companieshouse.filinghistory.consumer.mapper.pretransform;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.api.delta.DescriptionValues;
+import uk.gov.companieshouse.api.delta.FilingHistory;
+import uk.gov.companieshouse.filinghistory.consumer.transformrules.TransformerTestingUtils;
+import uk.gov.companieshouse.filinghistory.consumer.transformrules.functions.FormatDate;
+
+@ExtendWith(MockitoExtension.class)
+class ResolutionNodeMapperTest {
+
+    private static final ObjectMapper objectMapper = TransformerTestingUtils.getMapper();
+    private static final String CASE_START_DATE = "01/01/2010";
+    private static final String RES_TYPE = "res type";
+    private static final String DESCRIPTION = "description";
+    private static final String DATE = "02/02/2011";
+    private static final String RESOLUTION_DATE = "03/03/2012";
+
+    private ResolutionNodeMapper resolutionNodeMapper;
+    @Mock
+    private FormatDate formatDate;
+
+    @Mock
+    private FilingHistory delta;
+
+    @BeforeEach
+    void setUp() {
+        resolutionNodeMapper = new ResolutionNodeMapper(objectMapper, formatDate);
```

Comment:
Do we need this? The value can be assigned once in the declaration, assuming the mapper is stateless

Reply:
@cgregoryCH explained that this fixes a mock ordering issue